### PR TITLE
Bala/Change trigger for lighthouse and smoketest

### PR DESCRIPTION
**Pull request by balakrishna-deriv**
Changes:

-  Change trigger for lighthouse and smoketest to `workflow_run`

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by DerivFE** [comment]
| Name | Result |
| :--- | :------ |
| **Build status**  | Building 🔨 |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6796245467) |

**Comment by DerivFE** [comment]
| Name | Result |
| :--- | :------ |
| **Build status**  | Building 🔨 |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6796267905) |

File: .github/workflows/generate-preview-link.yml [hunk 1]
```diff
@@ -189,3 +192,9 @@ jobs:
                   number: ${{steps.pr_information.outputs.issue_number}}
                   message: ${{steps.generate_preview_url.outputs.comment || steps.generate_failure_comment.outputs.comment }}
                   recreate: true
+
+            - name: Upload PR information to artifact
+              uses: actions/upload-artifact@v2
+              with:
+                  name: 'pr-${{github.run_id}}'
+                  path: .pr
```
Comment (balakrishna-deriv) [inline .github/workflows/generate-preview-link.yml:200]: Uploads the artifacts so that it can be used in lighthouse and smoketest workflows

**Comment by DerivFE** [comment]
**Preview Link**: https://change-trigger-for-l.deriv-com-preview-links.pages.dev
| Name | Result |
| :--- | :------ |
| **Build status**  | Completed ✅ |
| **Preview URL**  | [Visit Preview](https://change-trigger-for-l.deriv-com-preview-links.pages.dev) |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6796267905) |

**Comment by DerivFE** [comment]
| Name | Result |
| :--- | :------ |
| **Build status**  | Building 🔨 |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6796352588) |

**Comment by DerivFE** [comment]
**Preview Link**: https://change-trigger-for-l.deriv-com-preview-links.pages.dev
| Name | Result |
| :--- | :------ |
| **Build status**  | Completed ✅ |
| **Preview URL**  | [Visit Preview](https://change-trigger-for-l.deriv-com-preview-links.pages.dev) |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6796352588) |

**Review by ali-hosseini-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**